### PR TITLE
[SCRUM-27][Deepface-verify] Setup and tested Deepface verify method w…

### PR DESCRIPTION
**Pull request by sprashanthv-dev**
- Installed DeepFace as dependency
- Tested basic face verification with static images

### CODE REVIEW

File: facial_recognition/app.py [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
 
+
 @app.route("/")
 def hello_world():
-    return "<p>Hello, World!</p>"
\ No newline at end of file
+    return "<p>Team Wombat - Facial Recognition Service :) </p>"
+
+
+# TODO: Change to POST method and get the image as input
```
Comment (allaboutmike) [inline facial_recognition/app.py:13]: Yes post for this call. Put it into a ticket.

**Review by grace-joh — APPROVED** [review]
first step done! LGTM~ 

@allaboutmike please don't hate us :)

File: facial_recognition/app.py [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+# TODO: What if deepface fails ? - Handle error case
+# TODO: Any specific response format? - Status code, messages etc.
```
Comment (allaboutmike) [inline facial_recognition/app.py:15]: Two choices here:
Use the HTTP status code to indicate the result. So 200 OK means good to go, 401 / 403 means unauthorized. OR use HTTP status codes for the request status itself (200 OK means I got your message and was able to run without error, and error codes mean I was not). Then you could put the result of the comparison into the body of the message using JSON

File: facial_recognition/app.py [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+
 @app.route("/")
 def hello_world():
-    return "<p>Hello, World!</p>"
\ No newline at end of file
+    return "<p>Team Wombat - Facial Recognition Service :) </p>"
+
+
+# TODO: Change to POST method and get the image as input
+# TODO: What if deepface fails ? - Handle error case
```
Comment (allaboutmike) [inline facial_recognition/app.py:14]: Error case is to throw a 500 error back to the client, which initiates the manual process.

**Pull request merged**